cheroot/connections.py

```diff
@@ -170,48 +171,57 @@ def expire(self):
             self._selector.unregister(sock_fd)
             conn.close()
 
-    def get_conn(self):
```

Comment:
Removal of this generates a failure @ https://github.com/cherrypy/cheroot/pull/311/checks?check_run_id=1451730559 because it's linked from Sphinx. To avoid it, you could add it to the ignore list @ `docs/conf.py`. See https://github.com/cherrypy/cheroot/blob/c99085a/docs/conf.py#L104-L111.